app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -347,6 +347,8 @@ class BrowserTabFragment : Fragment(), FindListener {
                 omnibarTextInput?.let {
                     it.setSelection(it.text.length)
                 }
+                // ensures the begin of the url is always displayed
```

Comment:
Thanks for making this PR!

Looking at the change, I see it's basically undoing the lines immediately above it. i.e., the lines above set the selection to end of the text, and your change sets the selection back to the start.

I think the previous functionality of setting the selection to the end can be removed entirely now (it was needed for a previous omnibar design).

Would you mind testing if the existing `setSelection` call is removed, do we still need your change to set it to 0 again? I suspect we can remove all `setSelection` calls now.

If you're happy to test that out and deleting the code if that looks good, then we can get your PR merged in.